Sources/ConcordiumSwiftSDK/Model/Account.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Base58Check
+
+fileprivate let accountAddressBase58CheckVersion: UInt8 = 1
+
+struct AccountAddress {
+    let bytes: Data // 32 bytes (Base58Check without version byte)
+    
+    init(base58Check: String) throws {
```

Comment:
I would suggest a couple of tests. One for the sunshine scenario, and one where the Grpc error is thrown. 
In addition to verifying the behavior, it would also document the intention (I am not quite sure what this base58 version refers to). 

Does the base58Check have some relevance to the caller? If so, I would suggest adding an argument label

Reply:
Good suggestion; done.